Flexible Learning/units_manage_add.php

```diff
@@ -160,10 +160,11 @@
     // UNIT OUTLINE
     $form->addRow()->addHeading(__m('Unit Outline'))->append(__m('The contents of this field are viewable to all users, SO AVOID CONFIDENTIAL OR SENSITIVE DATA!'));
 
+    $unitOutline = getSettingByScope($connection2, 'Flexible Learning', 'unitOutlineTemplate');
```

Comment:
Can you check the codebase for an example that uses $settingGateway->getSettingByScope(), rather than the function, since these will all one day get refactored.